AnimateApngs/src/main/java/dev/vendicated/aliucordplugs/animateapngs/AnimateApngs.java

```diff
@@ -30,14 +30,14 @@
 public class AnimateApngs extends Plugin {
     private void initApng(ImageView view, String mediaUrl, Integer w, Integer h) {
         final var url = mediaUrl
-                // Strip proxy but only if they're discord domains
-                .replaceFirst("https://images-ext-.*?\\.discordapp.net/external/.*?/https/(media|cdn)\\.discordapp\\.(net|com)", "https://cdn.discordapp.com")
-                // media server serves them as regular pngs, only cdn serves actual apngs
-                .replace("media.discordapp.net", "cdn.discordapp.com");
+                // Strip proxy but only if they're Discord domains.
+                .replaceFirst("https://images-ext-.*?\\.discordapp\\.net/external/.*?/https/(?:media|cdn)\\.discordapp\\.(?:net|com)", "https://cdn.discordapp.com")
+                // Media server serves them as regular PNGs, only CDN serves actual APNGs.
+                .replace("media\\.discordapp\\.net", "cdn.discordapp.com");
```

Comment:
I thought string.replace() took a literal substring and not a regex? isn't just replaceFirst/replaceAll regex?

Reply:
Yep, seems like I overlooked that part a bit; reverting those bits.